### ISSUE TITLE
Providing Bridge Mode

### ISSUE THREAD

**Issue by Knogle** [angristan/openvpn-install#549]
Ahoy. Everything is running fine.
Is there a easy way to provide bridge support instead of NAT? Maybe it's worth to add it, like having an option between NAT and Bridge mode?

**Comment by brnl** [comment]
You can configure OpenVPN to use the same network subnet as your LAN and forward DHCP, if that's what you mean? But I think it's better to just route the networks you need.

Can you explain your use case?

Maybe have a look at these pull requests:
https://github.com/angristan/openvpn-install/pull/550 Custom IP Range by @TheBritishAccent
https://github.com/angristan/openvpn-install/pull/547 New feature: Split-tunnel configuration by me

**Comment by Knogle** [comment]
Thanks for your reply.
So i want to assign ip addresses by the local network's DHCP to the OpenVPN clients.
I got an OpenWRT router as OpenVPN client, connected to my remote network, and there is a VoIP telephone connected to the OpenWRT router. So in order to use it, i have to bridge it into the remote network, by assigning an ip address by the DHCP to the telephone.
A similar procuedure i had to use for virtual machines, to assign ip addresses by the dhcp of the network. I had to create a network bridge. But i don't know if the problem can be resolved in the same way using this openvpn installer.
